ambry-router/src/test/java/com/github/ambry/router/TtlUpdateManagerTest.java

```diff
@@ -177,20 +179,28 @@ public void metadataChunkUpdatedLastTest() throws Exception {
     // configure failure for the metadata chunk
     serverLayout.getMockServers()
         .forEach(mockServer -> mockServer.setErrorCodeForBlob(blobIds.get(0), ServerErrorCode.Unknown_Error));
-    executeOpAndVerify(blobIds, RouterErrorCode.UnexpectedInternalError, false, false, false,
+    executeOpAndVerifyWithInfinity(blobIds, RouterErrorCode.UnexpectedInternalError, false, false, false,
         blobIds.subList(1, blobIds.size()));
   }
 
+  /**
+   * Test for the case when delete operation fails due to quota compliance.
+   */
+  @Test
```

Comment:
Is TtlUpdateManagerTest the only place where we are testing if operation fails due to quota compliance? Do we have such tests for Put, Delete, Undelete and Get as well?

Reply:
We do, it seems like only TtlUpdateOperation doesn't have such test. Please search on GetBlobOperationTest, UndeleteManagerTest, DeleteManagerTest and PutOperationTest file to search QuotaReject. you will find all these test files have some test cases that cover quota rejection error.